src/legacy/views/secondary-bar/parts/edit/share-folder-properties.tsx

```diff
@@ -65,19 +83,20 @@ const Actions = ({ folder, grant, createSnackbar, setActiveModal, onMouseLeave,
 			})
 		).then((res) => {
 			if (res.type.includes('fulfilled')) {
-				createSnackbar({
-					key: `resend-${folder.id}`,
-					replace: true,
-					type: 'info',
-					label: t('snackbar.share_resend', 'Share invite resent'),
-					autoHideTimeout: 2000,
-					hideButton: true
-				});
+				createSnackbar &&
+					createSnackbar({
+						key: `resend-${folder.id}`,
+						replace: true,
+						type: 'info',
+						label: t('snackbar.share_resend', 'Share invite resent'),
+						autoHideTimeout: 2000,
+						hideButton: true
+					});
 			}
 		});
 	}, [accounts, dispatch, folder, t, grant.d, createSnackbar]);
 	const onEdit = useCallback(() => {
-		setActiveGrant(grant);
+		setActiveGrant && setActiveGrant(grant);
```

Comment:
Prefer the shortened form 
```suggestion
		setActiveGrant?.(grant);
```